CHANGELOG.md

```diff
@@ -28,7 +28,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Added `ModelUsage` to `ModelCard` class.
 - Added a way to specify extra parameters to the predictor in an `allennlp predict` call.
-- Added a way to initialize a `Vocabulary` from transformers models. 
+- Added a way to initialize a `Vocabulary` from transformers models.
+- Add new method on `Field` class: `.human_readable_repr() -> object`
```

Comment:
```suggestion
- Add new method on `Field` class: `.human_readable_repr() -> Any`
```

Reply:
Didn't make the change before. Thanks for the scrutiny!!!